bin/layerlens.dart

```diff
@@ -27,18 +29,50 @@ enum _Options {
 
 void main(List<String> args) async {
   final parser = ArgParser()
-    ..addOption(_Options.path.name, defaultsTo: '.')
+    ..addFlag(
+      _Options.usage.name,
```

Comment:
I'm unsure about this parameter. General convention is to use `--help` which should be available by `args` package by default. However, that throws me an following exception:

```
Unhandled exception:
FormatException: Could not find an option named "help".
```